src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -762,6 +788,102 @@ where pattern.IsMatch(alias)
             return result;
         }
 
+        private static List<(string name, string description)> GetCommandDescriptions(string wordToComplete)
+        {
+            var context = LocalPipeline.GetExecutionContextFromTLS();
+            var helpRequest = new HelpRequest(wordToComplete, HelpCategory.Cmdlet | HelpCategory.Function)
+            {
+                ProviderContext = new ProviderContext(null, context, context.SessionState.Path)
+            };
+            var helpInfos = context.HelpSystem.GetHelp(helpRequest);
+            if (helpInfos == null)
+                return null;
```

Comment:
Please use curlies `{...}`.

Reply:
>Braces usually go on their own lines, with the exception of single line statements that are properly indented.

Reply:
The pattern is very dangerous.
In the repo we use 
```c#
if ( ... )
{
    ...
}
```
or
```c#
if ( ... ) return null;
```